ui/raidboss/data/04-sb/raid/o7s.txt

```diff
@@ -137,28 +137,28 @@ hideall "--sync--"
 #### Divergent path, Version A2: Dadaluma->Biblio
 
 # Dadaluma
-3509 "Load Dada" sync /:Guardian:(275C|2773):/
-3512 "Shockwave" sync /:Guardian:2783:/
+3509 "Load Dada" sync / 1[56]:Guardian:(275C|2773):/
+3512 "Shockwave" sync / 1[56]:Guardian:2783:/
 3516 "Radar" duration 2
-3522 "Missile Simulation" sync /:Guardian:2764:/
-3531 "Atomic Ray" sync /:Guardian:278D:/
-3536 "Chakra Burst" sync /:Guardian:2787:/
-3548 "Prey" sync /:Guardian:278A:/
-3554 "Magitek Ray" sync /:Guardian:2788:/
-3562 "Diffractive Plasma" sync /:Guardian:278B:/
+3522 "Missile Simulation" sync / 1[56]:Guardian:2764:/
+3531 "Atomic Ray" sync / 1[56]:Guardian:278D:/
+3536 "Chakra Burst" sync / 1[56]:Guardian:2787:/
+3548 "Prey" sync / 1[56]:Guardian:278A:/
+3554 "Magitek Ray" sync / 1[56]:Guardian:2788:/
+3562 "Diffractive Plasma" sync / 1[56]:Guardian:278B:/
 
 # Bibliotaph
-3572 "Load Biblio" sync /:Guardian:(275C|2774):/
+3572 "Load Biblio" sync / 1[56]:Guardian:(275C|2774):/
 3581 "Radar" duration 2
-3582 "Demon Simulation" sync /:Guardian:2B36:/
-3595 "Run Biblio" sync /:Guardian:276F:/
-3602 "Diffractive Plasma" sync /:Guardian:278B:/
-3609 "Diffractive Plasma" sync /:Guardian:278B:/
-3616 "Diffractive Plasma" sync /:Guardian:278B:/
-3624 "Prey" sync /:Guardian:278A:/
-3633 "Magitek Ray" sync /:Guardian:2788:/
-3642 "Diffractive Plasma" sync /:Guardian:278B:/
-3648 "--sync--" sync /:(275C|2773):Guardian/ jump 648          # Load/Skip starts casting, return to A
+3582 "Demon Simulation" sync / 1[56]:Guardian:2B36:/
+3595 "Run Biblio" sync / 1[56]:Guardian:276F:/
+3602 "Diffractive Plasma" sync / 1[56]:Guardian:278B:/
+3609 "Diffractive Plasma" sync / 1[56]:Guardian:278B:/
+3616 "Diffractive Plasma" sync / 1[56]:Guardian:278B:/
+3624 "Prey" sync / 1[56]:Guardian:278A:/
+3633 "Magitek Ray" sync / 1[56]:Guardian:2788:/
+3642 "Diffractive Plasma" sync / 1[56]:Guardian:278B:/
+3648 "--sync--" sync / 14:Guardian/ jump 648          # Load:(275C|2773):/ jump 648          # Load
```

Comment:
:eyes:

(here, and in at least two other spots)

Reply:
Aaaaaaa!